multicluster/config/crd/bases/multicluster.x-k8s.io_serviceexports.yaml

```diff
@@ -0,0 +1,133 @@
+# Copyright 2020 The Kubernetes Authors.
```

Comment:
i don't think we put copyright header in yaml files.. at least don't see that in rest of antrea build/yamls

Reply:
ok, let me remove it.